src/api/init.ts

```diff
@@ -38,6 +38,11 @@ export async function startApiServer(datastore: DataStore): Promise<ApiServer> {
 
   const apiHost = process.env['STACKS_BLOCKCHAIN_API_HOST'];
   const apiPort = parseInt(process.env['STACKS_BLOCKCHAIN_API_PORT'] ?? '');
+  const dev = process.env.NODE_ENV !== 'production';
+
+  const { createMiddleware: createPrometheusMiddleware } = require('@promster/express');
```

Comment:
Modules should be imported using esm syntax. This needs to be something like: 
`import { createMiddleware as createPrometheusMiddleware } from '@promster/express';`
and located at the top of the file alongside the other imports.

Reply:
@zone117x [This is what I tried previously here](https://github.com/blockstack/stacks-blockchain-api/pull/238/commits/ff8f620aaff5040dd166c31f2b902124eef3a10f#diff-f0977764d44e25489d36496dcb219a87R26-R27), but it resulted in various errors when trying to use the module. Not sure if there is a functional difference between the syntax I used and your example. The promster documentation for using `import` is lacking and they seem to be very specific about using `require`, so that's why I changed it back. Will look into it more today and try your syntax.